### ISSUE TITLE
'gnt-cluster copyfile' is not using dedicated network

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#88]
Originally reported of Google Code with ID 32.

```
I just discovered, that when I do a 'gnt-cluster copyfile' to copy a file
over a cluster with a dedicated, secondairy network for remote_raid1,
ganeti still uses the normal, public network. Is this known issue? Perhaps
you could make an extra option to use a different network in a future release.

Specs:
- Debian Etch
- Ganeti 1.2.4
```
Originally added on 2008-07-04 13:11:57 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Hi,

Yes, by design every traffic except the replication traffic goes over the public
network, but is encrypted and authenticated.

For example, 'gnt-cluster copyfile' uses scp to copy the files.

Do you think it would be worth to allow using the private network too?

thanks,
iustin
```
Originally added on 2008-07-04 13:17:33 +0000 UTC.

Added Labels: Type-Enhancement
Removed Labels: Type-Defect

**Comment by Ganeti-Issues-Migrator** [comment]
```
For me it would quite a nice option, because all the traffic on the 'normal' network
is counted and billed. Traffic on the lan-switch is free.
```
Originally added on 2008-07-04 13:27:50 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Ah.... it's interesting that because we use ganeti in a different environment,
we didn't think of this point. That's the good point of opensource :)

Thank you, I agree that it's a very good use case. We'll look at implementing
it.

```
Originally added on 2008-07-04 13:30:49 +0000 UTC.

Added Labels: Priority-Low
Removed Labels: Priority-Medium
Changed State: Accepted
Added to Milestone: Release2.0

**Comment by Ganeti-Issues-Migrator** [comment]
```
That would be really great! I'm looking forward for this feature!
```
Originally added on 2008-07-04 15:13:35 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2009-08-10 08:16:41 +0000 UTC.

Added to Milestone: Release2.1
Removed from Milestone: Release2.0

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2009-09-16 12:48:52 +0000 UTC.

Added to Milestone: Release2.2
Removed from Milestone: Release2.1

**Comment by Ganeti-Issues-Migrator** [comment]
```
This has been implemented and will be released in version 2.1.2; sorry for the late 
reply...

iustin
```
Originally added on 2010-03-17 16:29:59 +0000 UTC.

Changed State: Fixed
Added to Milestone: Release2.1
Removed from Milestone: Release2.2

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-22 08:57:07 +0000 UTC.

Changed State: Released
